pkg/query/api/v1.go

```diff
@@ -178,6 +183,8 @@ type queryData struct {
 	Warnings []error `json:"warnings,omitempty"`
 }
 
+type storesData map[string][]query.StoreStatus
```

Comment:
This change was made by Adrien so I can just guess. What I understand is `api.stores` returns a map with Store's Type as `key` and array of the store's statuses as it's `value`, while `api.storeSet.GetStoreStatus()` returns a flat array of statuses.

I guess this was done because it would make sense from the perspective of an api client, as the current approach would give the client a json object with Store's name as key and an array of status as it's value, which is easier to consume imo.

Reply:
Yes, it's all good with the actual type but we can convert:

```golang
638     statuses := make(storesData)                                                                                                                                                                               
```

Into:

```golang
638     statuses := make(map[string][]query.StoreStatus)
```

There's no need to have that type alias since there is nothing else consuming it. I think it's a case of YAGNI: https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it. If we will need it then we can always add it back.

Reply:
Oh got it :+1: I'll update the PR.

Reply:
This was solved so marking as 'Resolved'.